tests/app/views/contexts/summary/test_answer.py

```diff
@@ -6,20 +6,24 @@
 
 @pytest.mark.usefixtures("app")
 @pytest.mark.parametrize(
-    "return_to, return_to_block_id, is_in_repeating_section",
+    "return_to, return_to_block_id, is_in_repeating_section, return_to_answer_id",
     [
-        ("section-summary", None, False),
-        (None, None, False),
-        ("calculated-summary", "total", False),
-        ("section-summary", None, True),
-        (None, None, True),
-        ("calculated-summary", "total", True),
+        ("section-summary", None, False, None),
+        (None, None, False, None),
+        ("calculated-summary", "total", False, None),
+        ("section-summary", None, True, None),
+        (None, None, True, None),
+        ("calculated-summary", "total", True, None),
+        ("calculated-summary", "total", True, "calculated-summary-1"),
     ],
 )
-def test_create_answer(return_to, return_to_block_id, is_in_repeating_section):
+def test_create_answer2(
```

Comment:
I think you might have meant to change the function name back to not having the `2`

Reply:
Done, thanks.